modules/meta/arrow-meta-prototype/idea-plugin/src/main/kotlin/arrow/meta/plugin/idea/phases/resolve/MetaSyntheticPackageFragmentProvider.kt

```diff
@@ -205,7 +131,6 @@ class MetaSyntheticPackageFragmentProvider :
   }
 
   inner class BuildCachePackageFragmentDescriptor(module: ModuleDescriptor, fqName: FqName) : PackageFragmentDescriptorImpl(module, fqName) {
```

Comment:
does this has to be an inner class or can we move this to an object and pass it implicitly